node-admin/src/main/application/services.xml

```diff
@@ -4,6 +4,10 @@
   <jdisc id="node-admin" jetty="true" version="1.0">
     <!-- Please update container test when changing this file -->
     <accesslog type="vespa" fileNamePattern="logs/vespa/node-admin/access.log.%Y%m%d%H%M%S" rotationScheme="date" symlinkName="access.log" />
+    <handler id="com.yahoo.vespa.hosted.node.admin.restapi.RestApiHandler" bundle="node-admin">
```

Comment:
The RestApiHandler is really not a good fit for the host admin rest api since resume & suspend should be internal to host admin.

How about making a host-admin rest api handler that only supports metrics for now. this will also avoid dependency on NASU in the handler.